pkg/annotation/annotation.go

```diff
@@ -0,0 +1,13 @@
+// Package annotation contains common Kubernetes metadata. These are defined in
+// https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md.
+package annotation
+
+const (
+	// CordonReasonAnnotationName is the name of the annotation that indicates
+	// the reason of why chart-operator should not apply any update on this chart CR.
+	CordonReasonAnnotationName = "app-operator.giantswarm.io/cordon-reason"
```

Comment:
```suggestion
	CordonReasonAnnotationName = "chart-operator.giantswarm.io/cordon-reason"
```

Reply:
Should be named this way because the annotations are consumed by chart-operator.